### PR TITLE
feat(pages): consuming get-producers-info api

### LINKED ISSUE #150: Get bpjsons info online
- [ ] Develop API
- [ ] Consume the API

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR have?
- Resolve #150

### CODE REVIEW

File: src/utils/bps.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const getBpJons = async (owners: Array<string>): Promise<Array<any>> => {
+  try {
+    const {
+      getProducersInfo: { producersInfo }
+    } = await (
+      await fetch(
+        'https://graphql-eos.antelope.tools/api/rest/get-producers-info',
```
Comment (xavier506) [inline src/utils/bps.ts:7]: lets define this as an env var

**Comment by AngeloCG97** [comment]
Please don't merge, we are  review producers api response

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**